src/components/discover-sheet/TopMoversSection.js

```diff
@@ -73,14 +91,20 @@ export default function TopMoversSection() {
         </Flex>
       )}
 
-      <Column>
-        {gainerItems?.length !== 0 && (
-          <MarqueeList items={gainerItems} speed={40} />
-        )}
-        {loserItems?.length !== 0 && (
-          <MarqueeList items={loserItems} speed={-40} />
-        )}
-      </Column>
+      {network !== networkTypes.mainnet ? (
+        <ErrorMessage colors={colors}>
+          Top movers are disabled on Testnets
```

Comment:
I don't have a strong opinion, but I think that:
1. In someone is using testnet, then is expecting that some things might not work 
2. Then those explicit warnings might be redundant and we can just hide sections

Reply:
If I hide everything Then we're left with an entire blank sheet.

If you wanna make discover sheet permanently shorten when on testnets (showing only the scanner) it's ok with me but I'm don't think we should hold the release because of this.

Reply:
Ah, it hides basically everything but for lists? And DPI? Yeah, you are right

Reply:
Agreed we should disable the entire sheet but agree its a non blocker

Reply:
Let us leave it in the current state. I might only have an opinion that only top movers can be just hidden. We do it anyway if we receive no response from zerion. But if this blocks anything, then idc!